config/src/lib.rs

```diff
@@ -110,6 +112,10 @@ impl ConnectionsConfig {
     pub fn default_known_peers() -> Vec<SocketAddr> {
         Vec::default()
     }
+
+    fn default_storage_peers_period() -> u64 {
+        30
```

Comment:
This is a magic number, please:
- Move it to a more visible spot
- Make it static
- Add a comment for it, potentially reasoning about considerations when deciding the value

Reply:
and is it appropriate to mark this method `#[inline}`?

Reply:
Sounds good! :+1: